### PR TITLE
Fixed inline comment

### PR DESCRIPTION
**Pull request by Oleksijt**
There was wrong inline comment in my file.

### CODE REVIEW

File: oleksijt/task4_define_a_base_class_for_auto.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def __init__(self, car_type, model, year, max_speed):
 
     def change_type(self, car_type):
         """ Change car_type. """
-        self.car_type = car_type  # Set new type for changed class.
+        self.car_type = car_type  # Set new type.
```
Comment (pivancy) [inline oleksijt/task4_define_a_base_class_for_auto.py]: It's better to write comments in this way:

# Your comment goes here.
<block of code>

Comment (Oleksijt) [reply oleksijt/task4_define_a_base_class_for_auto.py]: What way?
I only see the text "Your comment goes here."

Comment (pivancy) [reply oleksijt/task4_define_a_base_class_for_auto.py]: Sorry, updated.

Comment (Oleksijt) [reply oleksijt/task4_define_a_base_class_for_auto.py]: Fixed.

**Comment by pivancy** [comment]
👍

**Pull request merged**